src/app/legacy/containers/CpsOnwardJourney/index.jsx

```diff
@@ -281,7 +281,7 @@ CpsOnwardJourney.defaultProps = {
   parentColumns: null,
   sectionLabelOverrideAs: null,
   sectionLabelBar: true,
-  sectionLabelBackground: C_GHOST,
+  sectionLabelBackground: GHOST,
```

Comment:
Was not sure the best to use theme in default props

Reply:
This looks fine